google-oauth-client-jetty/pom.xml

```diff
@@ -89,6 +89,7 @@
     <dependency>
       <groupId>org.eclipse.jetty</groupId>
       <artifactId>jetty-server</artifactId>
+      <version>9.4.26.v20200117</version>
```

Comment:
versions are set elsewhere.

Reply:
I'm new to this.  How are versions set?  There are a number of people who need this upgrade (see the Issue).

Reply:
OK - I found where the versions are set and I will update my fix appropriately.  Is a new pull request appropriate or will it be rejected because the upgrade is not wanted (why not)?

Reply:
Jetty is a weird dep and we need to keep it consistent across the orbit of all GCP libraries, not just in this one project. Upgrading it to a new major version is going to require some planning and investigation to make sure we don't accidentally break anything. If this is necessary for some reason, please file an issue with more details so this can be prioritized and discussed.